Makefile

```
@@ -27,6 +27,7 @@ validate: registry
 $(plugins_dir):
 	mkdir -p $(plugins_dir)
 	chmod 700 $(plugins_dir)
+	chmod 700 ~/.op
```

I'm seeing the following error in certain instances when initializing local plugins using the Makefile in this PR:

```
[ERROR] 2022/12/19 08:45:22 Can't continue. We can't safely access "~/.op/plugins" because its permissions are too broad. Change its permissions to 700 and try again.
```

```suggestion
	chmod 700 ~/.op
	chmod 700 ~/.op/plugins
```

---

Adjusted :)